docker/prepare_proxy_test.sh

```diff
@@ -1,16 +1,57 @@
-#!/bin/sh
+#!/bin/bash
+# in glcoud: prepare_proxy_test.sh 
+# in your dev environment: docker/prepare_proxy_test.sh -d "quay.io" -p local -c 192.168.0.0/16
+#
 
 set -o errexit
 set -o nounset
 set -o pipefail
 set -ex
 
-# customize for development / debug setup
+usage () {
+  echo "${0} [-d HUB] [-p gcloud|local] [-c cidrs] [-h]"
+  echo ''
+  echo ' -d: docker hub to use (default: docker.io/) (optional)'
+  echo ' -p: platform to use (default: gcloud)  (optional)'
+  echo ' -c: list of cidrs to use (default: self selected) (optional)'
+  echo ''
+}
+
+LIST_CIDRS=""
 HUB=docker.io/$(whoami)
+PLATFORM=gcloud
+
+while getopts ":d:p:c:h" opt; do
+  case ${opt} in
+    d)
+      HUB=${OPTARG}/$(whoami)
+      ;;
+    p)
+      PLATFORM=${OPTARG}
+      if [ "${PLATFORM}" != "gcloud" ] && [ "${PLATFORM}" != "local" ]; then
+        echo "Invalid platform ${PLATFORM}" >&2
+        usage
+        exit 1
+      fi
+      ;;
+    c)
+      LIST_CIDRS=${OPTARG}
+      ;;
+    h)
+      usage
+      exit 0
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      usage
+      exit 1
+      ;;
+  esac
+done
+
 TAG=test
```

Comment:
Same comment - please allow env overrides ( ${VAR:-default} ), so others can run the test with different settings.